QModManager/API/SMLHelper/Utility/ReflectionHelper.cs

```diff
@@ -0,0 +1,192 @@
+﻿namespace QModManager.API.SMLHelper.Utility
+{
+    using System.Diagnostics;
+    using System.Reflection;
+
+    /// <summary>
+    /// An extension method class for simplifying reflection calls for improved readability.
+    /// </summary>
+    public static class ReflectionHelper
```

Comment:
Has lost its purpose

Reply:
For us, maybe.  
But this was also offered as a library for other people to use if they wanted.  
If we remove it, we should keep the code somewhere accessible as a reference tool for others.

Reply:
It's already in QModManager...